drivers/console/Kconfig

```diff
@@ -108,10 +108,9 @@ config USB_UART_CONSOLE
 	select USB_CDC_ACM
 	help
 	  Enable this option to use the USB UART for console output. The output
-	  can be viewed from the USB host via /dev/ttyACM* port. Note that console
-	  inputs from the USB UART are not functional yet. Also since the USB
-	  layer currently doesn't support multiple interfaces, this shouldn't be
-	  selected in conjunction with, say, USB Mass Storage.
+	  can be viewed from the USB host via /dev/ttyACM* port.
+	  As for the console driver, this option only changes the initialization
+	  level.
```

Comment:
It's not only about the console output. The console communication works bidirectional. Why not rephrase the first two sentences like this:
`Enable this option to use the USB UART for the console. The communication is done via /dev/ttyACM* virtual port.`

Reply:
okay, I have now removed "output"